File_Adapter/AdapterActions/Push.cs

```diff
@@ -60,14 +57,9 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
 
             CreateFileAndFolder(pushType);
 
-            if (objectsToPush.Count() != objects.Count())
```

Comment:
same here, should this really be removed now that we are keeping the pre-processing as before?